coordinator/src/db/trades.rs

```diff
@@ -24,6 +24,7 @@ struct Trade {
     average_price: f32,
     timestamp: OffsetDateTime,
     fee_payment_hash: String,
+    expiry_timestamp: Option<OffsetDateTime>,
```

Comment:
❓ what is the expiry on the trade?

Reply:
As discussed privately, this expiry corresponds to the expiry that needs to be used to set up the new DLC channel. It clearly does not belong in the `Trade` model, but it was a convenient thing to do to avoid creating a new table.